filebeat/module/system/syslog/ingest/pipeline.json

```diff
@@ -36,10 +36,18 @@
                     "MMM dd HH:mm:ss",
                     "yyyy-MM-dd'T'HH:mm:ss.SSSSSSZZ"
                 ],
-                {< if .convert_timezone >}"timezone": "{{ event.timezone }}",{< end >}
                 "ignore_failure": true
             }
         },
+        {
```

Comment:
Do we need to apply this change to all other filebeat modules that currently have the `< if .convert_timezone >}"timezone"` check?

Reply:
Would be great to have a follow up on other parts.

Reply:
Yes, once we agree on this we can apply the change to the rest of modules.

Reply:
Follow up for the rest of pipelines: https://github.com/elastic/beats/pull/12311